Dockerfile.processing_image

```diff
@@ -14,14 +14,15 @@ ENV PIP_NO_CACHE_DIR=1
 RUN git clone https://github.com/chanzuckerberg/single-cell-explorer.git \
  && echo '#!/bin/sh\nmkdir -p build/static && touch build/static/.gitignore build/index.html build/csp-hashes.json' > /usr/bin/npm && chmod 755 /usr/bin/npm \
  && cd single-cell-explorer \
- && make pydist-czi-hosted \
+ && make pydist-server \
  && make install-dist \
- && make dev-env-czi-hosted \
+ && make dev-env-server \
  && cd .. \
  && rm -rf single-cell-explorer client build/client /usr/bin/npm
 
 # Install python dependencies
-RUN pip3 install loompy==3.0.6 scanpy==1.6.0 python-igraph==0.8.3 louvain==0.7.0 tiledb==0.7.4 cellxgene-schema awscli
+RUN pip3 install loompy==3.0.6 scanpy==1.6.0 python-igraph==0.8.3 louvain==0.7.0 tiledb==0.7.4 awscli
+RUN pip3 install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple cellxgene-schema==1.0.2
```

Comment:
TODO: Update to prod pip & cellxgene-schema version when ready

Reply:
Shall we upload to prod pip today, or do we want to at least run a full E2E testing before we do that? @pablo-gar

Reply:
We should do the testing before